tests/data/test_template_tokenization.py

```diff
@@ -163,3 +163,23 @@ def test_tokenize_instruct_example_well_formed():
             tokenized_example = tokenize_formatted_example(example, tokenizer)
             assert 'input_ids' in tokenized_example
             assert 'labels' in tokenized_example
+
+
+def test_tokenize_no_labels_bos():
+    # This tokenizer automatically adds bos tokens
+    tokenizer = transformers.AutoTokenizer.from_pretrained(
+        'mistralai/Mixtral-8x7B-v0.1')
+
+    example = {'prompt': 'prompt', 'response': 'response'}
+
+    tokenized_example = tokenize_formatted_example(example, tokenizer)
+
+    assert len(tokenized_example['labels']) == 1
+    assert tokenized_example['labels'][0] != '<s>'
```

Comment:
`tokenized_example['labels']` produces a list of int. How about comparing against tokenizer.bos_token_id

```suggestion
    assert len(tokenized_example['labels']) == 1
    assert tokenized_example['labels'][0] != tokenizer.bos_token_id
    assert len(tokenized_example['input_ids'] == 2
    assert tokenized_example['input_ids'][0] == tokenizer.bos_token_id
```

Reply:
oops, thanks